core-api/src/main/java/com/optimizely/ab/event/internal/payload/Conversion.java

```diff
@@ -192,6 +198,7 @@ public String toString() {
                 ", eventMetrics=" + eventMetrics +
                 ", eventFeatures=" + eventFeatures +
                 ", isGlobalHoldback=" + isGlobalHoldback +
+                ", anonymizeIP=" + anonymizeIP +
```

Comment:
Yeah we should at some point since we share these with `Impression.java`. Just don't think it's worth refactoring for this diff :)